app/javascript/components/patient/PatientPage.js

```diff
@@ -10,7 +10,7 @@ class PatientPage extends React.Component {
   render() {
     return (
       <Card id="patient-page" className="mx-2 my-4 card-square">
-        <Card.Header className="h5" id="patient-info-header">
+        <Card.Header as="h1" className="patient-card-header" id="patient-info-header">
```

Comment:
We're gonna want this same change in `Review.js` here: https://github.com/SaraAlert/SaraAlert/blob/d9c79d31e40b5d11fd584e45455625febf3540f7/app/javascript/components/enrollment/steps/Review.js#L65

Reply:
fixed

Reply:
This is not fixed. The HTML it renders:
![image](https://user-images.githubusercontent.com/8235974/127004636-1ad9db27-6019-4efd-b50e-9a6f012afb0f.png)